terraform/groups/ecs-service/profiles/staging-eu-west-2/staging/vars

```diff
@@ -2,7 +2,7 @@ environment = "staging"
 aws_profile = "staging-eu-west-2"
 
 # service configs
-use_set_environment_files = true
+log_level = "info"
 
 # scaling configs
```

Comment:
```suggestion
# Scheduled scaling of tasks
```
Minor point, choose between scaling configs or Scheduled scaling of tasks for consistency across vars